lib/kinetic_cafe/error.rb

```diff
@@ -25,7 +25,7 @@ module KineticCafe # :nodoc:
   # rescue clause and handled there, as is shown in the included
   # KineticCafe::ErrorHandler controller concern for Rails.
   class Error < ::StandardError
-    VERSION = '1.8.1' # :nodoc:
+    VERSION = '1.9' # :nodoc:
```

Comment:
There’s only performance benefits if people are working with KineticCafe::Error::VERSION. Other strings may be worthwhile freezing, but I’m not sure that there’s a lot of them to freeze here.